Realm/RLMObjectBase.mm

```diff
@@ -488,6 +488,18 @@ id RLMObjectFreeze(RLMObjectBase *obj) {
     return frozen;
 }
 
+id RLMObjectThaw(RLMObjectBase *obj) {
+    RLMVerifyAttached(obj);
```

Comment:
There is no check for whether this is object is managed because it couldn't be unmanaged if it were previously frozen. If it were not frozen, then the method returns.